buildconfig/Jenkins/buildscript

```diff
@@ -183,29 +183,22 @@ if [[ "$BUILDPKG" == true ]]; then
   # Set some variables relating to the linux packages
   if [[ ${ON_MACOS} != true ]]; then
     # Use different suffix for linux builds
-    if [[ ${JOB_NAME} == *release* ]]; then
-      echo "Performing release build" # do nothing
-    elif [[ ${JOB_NAME} == *master* ]]; then
+    if [[ ${JOB_NAME} == *master* ]]; then
       PACKAGE_SUFFIX="nightly"
     elif [[ ${JOB_NAME} == *pvnext* ]]; then
       PACKAGE_SUFFIX="mantidunstable-pvnext"
     else
       PACKAGE_SUFFIX="unstable"
     fi
 
-    # Add '-python3' to package name and install path
-    if [[ ${JOB_NAME} == *python3* ]]; then
-	PACKAGE_SUFFIX=${PACKAGE_SUFFIX}-python3
-    fi
-
-    # Only unsuffixed release builds create envvars scripts
-    if [[ ${JOB_NAME} == *release* && -z "$PACKAGE_SUFFIX" ]]; then
-      PACKAGINGVARS="${PACKAGINGVARS} -DENVVARS_ON_INSTALL=True -DCPACK_SET_DESTDIR=ON"
+    if [[ ${JOB_NAME} == *release* ]]; then
+      # No suffix and traditional install path
+      PACKAGINGVARS="${PACKAGINGVARS} -DCMAKE_INSTALL_PREFIX=/opt/Mantid -DCPACK_PACKAGE_SUFFIX="
```

Comment:
If we are having coexisting python 2 and 3 releases soon won't the packages need unique names?

Reply:
Good point - in my head python 3 branch went through the else branch and became `mantid-python3` but of course that will have `release` in the name too. I'll fix that.